app.css

```diff
@@ -2,45 +2,101 @@ html {
 	background-color: #2f4f4f;
 }
 
-.red {
+body {
+	font-family: 'PT Mono';
+	color: #2E3D37
+}
+
+.my-red {
 	color: red;
-	font-size: 1.3em;
 }
 
-.grey {
-	color: #e9e849;
+.my-grey {
```

Comment:
I had to change the names because materialize defines classes called red and grey. I didn't need to do this with underline, but I did for consistency.